src/hooks/useActiveOrgV2.tsx

```diff
@@ -0,0 +1,59 @@
+import { getActiveOrgId, setActiveOrgId as setActiveOrgIdCookie } from "@propelauth/javascript"
+import { useContext, useEffect, useState } from "react"
+import { AuthContext } from "../AuthContext"
+
+const ACTIVE_ORG_ID_LOCAL_STORAGE_KEY = "__ACTIVE_ORG_ID"
+
+interface ActiveOrg {
+    activeOrgId?: string
```

Comment:
This type should be an OrgMemberInfo instead of an id

Reply:
Changed to the `OrgMemberInfoClass`